localstack/services/apigateway/helpers.py

```diff
@@ -667,13 +672,16 @@ def add_path_methods(rel_path: str, parts: List[str], parent_id=""):
 
         paths_dict = resolved_schema["paths"]
         method_paths = paths_dict.get(rel_path, {})
-        for method, method_schema in method_paths.items():
-            method = method.upper()
+        for field, field_schema in method_paths.items():
+            if field in ["parameters", "servers", "description", "summary", "$ref"]:
```

Comment:
Great that we're filtering out the values here. 👍 

If we want to be on the safe side, we could additionally check for `dict` instances here:
```
if not isinstance(field_schema, dict):
    LOG.warning(...)
    continue
```